brd-ios/breadwallet/src/Redesign/Scenes/Swap/AssetSelection/AssetSelectionStore.swift

```diff
@@ -16,6 +16,7 @@ class AssetSelectionStore: NSObject, BaseDataStore, AssetSelectionDataStore {
     var supportedCurrenciesPair: [String]?
     var isFromCurrency: Bool?
     var baseCurrencySelected: Currency?
+    var termCurrencySelected: Currency?
```

Comment:
do u need 2 properties?
is just "selected" not enough, to show what can be selected?
(i presume that even if BSV is term currency.. on base we still support selecting it, right?)